runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -5235,11 +5239,23 @@ class INTERPRETER_CLASS
 				UDATA result = preparePinnedVirtualThreadForUnmount(_currentThread, object, true);
 				VMStructHasBeenUpdated(REGISTER_ARGS);
 				if (J9_OBJECT_MONITOR_OOM != result) {
+					PORT_ACCESS_FROM_JAVAVM(_vm);
 					/* Handle the virtual thread Object.wait call.
 					 * VirtualThread.timeout is a private field used by both VM and JCL to temporarily hold
 					 * the value of expected wait/park time before a wake up task is scheduled using the value.
+					 *
+					 * Note:
+					 * For VirtualThread.park() the timeout is measured in nanoseconds.
+					 * For Object.wait(...) the timeout is measured in milliseconds,
+					 * and any non-zero nanosecond value always roundup to 1 millisecond.
 					 */
```

Comment:
Why do the comments mention "park" here at all, particularly after it says "Handle the virtual thread Object.wait call"?

Reply:
without reference to nanosecond used by park, I don't see the reason for a comment to specifically pointing out why use of millisecond would need clarification to prevent future confusion as suggested in https://github.com/eclipse-openj9/openj9/pull/21590#discussion_r2040301390

Reply:
Ok, it's fair to refer to the differing expectations for `park()`, but I still the comments could be improved, even if only to qualify "Object.wait(...)" with words that convey that that is the situation being dealt with here.

Reply:
Even replacing this comment with a copy of the comment on line 5825 would be better.

Reply:
okay, will update the comment to match 5825